cstar/entrypoint/worker/worker.py

```diff
@@ -165,7 +164,7 @@ def _prepare_file_system(self) -> None:
             None,
         )
 
-        if self._output_root.exists() and outputs:
+        if self._output_dir.exists() and outputs:
```

Comment:
I am convinced this method shouldn't be on this request and should be done by the code handling it. It's not your problem, but maybe we should create a follow-up ticket to just shift it, later.

Reply:
per offline convo, 👍